fmriprep/workflows/bold/base.py

```diff
@@ -791,11 +857,19 @@ def _bold_reg_suffix(fallback, freesurfer):
         (inputnode, ds_bold_rois_report, [('source_file', 'source_file'),
                                           ('bold_rois_report', 'in_file')]),
         (inputnode, ds_bold_reg_report, [
-            ('source_file', 'source_file'),
             ('bold_reg_report', 'in_file'),
             (('bold_reg_fallback', _bold_reg_suffix, freesurfer), 'suffix')]),
     ])
 
+    if t2s_coreg:
+        workflow.connect([
```

Comment:
maybe worth collapsing this if..else into a one liner with the main connect block we had previously:

```Python
workflow.connect([
    # ... connections
    (inputnode, ds_bold_reg_report, [
        ('first_echo' if t2s_coreg else 'source_file', 'source_file')]),
    # ... connections
])
```

Reply:
I didn't know you could do that ! 🌟